input/images-source/localization-cardiologist-search.plantuml

```diff
@@ -9,19 +9,18 @@ hide footbox
 actor "Dr. Smith\n(Cardiologist)" as doctor
 participant "EHR System @\n(Care Provider, URA:456)" as ehr
 participant "Localization Service" as nvi
-participant "PACS System @\n(Care Provider, URA:123)" as pacs
+participant "EHR @\n(Care Provider, URA:123)" as ehr_reg
 
-doctor -> ehr: Request imaging data\n for patient (BSN:987654321)
+doctor -> ehr: Request data\n for patient (BSN:987654321)
```

Comment:
Who's BSN is this?
There is a test-range starting with 99999, better to use that.